helpers/mdxComponents.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import CodeBlock from '../components/mdx/CodeBlock'
+import BlockQuote from '../components/mdx/BlockQuote'
+import ModalImage from '../components/mdx/ModalImage'
+import styles from '../scss/mdx.module.scss'
+
+const MDXcomponents = {
```

Comment:
This maps markdown tags to custom components. So when you type ```code``` in markdown CodeBlock component will be rendered and etc.